managers/backup-manager/BackupManager.js

```diff
@@ -0,0 +1,489 @@
+'use strict';
```

Comment:
I guess we do not need all the methods here, looks like some unneeded methods are here.

Reply:
Removed